### ISSUE TITLE
Fail to running on multi-gpus 

### ISSUE THREAD

**Issue by inspurasc** [tensorflow/nmt#346]
ENV:
os:ubuntu16.4
python: 2.7.5
tensorflow:1.8.
cuda:9.2

While training WMT German-English, I request 4 gpus, but it running on 1 gpu. The cmd is below:

CUDA_VISIBLE_DEVICES=0,1,2,3 python -m nmt.nmt --src=de --tgt=en --hparams_path=nmt/standard_hparams/wmt16_gnmt_4_layer.json --out_dir=/tmp/deen_gnmt --vocab_prefix=/data/xinyx/wmt_data/vocab.bpe.32000 --train_prefix=/data/xinyx/wmt_data/train.tok.clean.bpe.32000 --dev_prefix=/data/xinyx/wmt_data/newstest2013.tok.bpe.32000 --test_prefix=/data/xinyx/wmt_data/newstest2015.tok.bpe.32000 --num_gpus=4  --num_train_steps=12000 --steps_per_stats=100 --num_layers=4 -num_units=128 --dropout=0.2 --metrics=bleu

Please give me some advice how i should do so that it can run on multiple GPU.
![bug](https://user-images.githubusercontent.com/19197677/40965162-8edde972-68df-11e8-9a11-66b1d947eca4.png)

**Comment by inspurasc** [comment]
I have solve this problem.Thanks.

**Comment by baihongx** [comment]
> I have solve this problem.Thanks.

can you give your solution? gracias

**Comment by mohammedayub44** [comment]
@inspurasc  How did you solve this problem if you can please post the solution.

Mohammed Ayub

**Comment by swinalmeida** [comment]
@inspurasc How did you resolve this issue. Can you please help!!
